### ISSUE TITLE
[BUG] There is always a padding when displaying content inside the editor (Hint text padding)

### ISSUE THREAD

**Issue by gerken-tss** [the-airbender/quill_html_editor#35]
**Describe the bug**
It doesn't matter which HTML content I render, it can even be a plain text, the editor always displays a padding of 8 pixels around the content. Needless to say, I didn't fiddle with the `padding` parameter of `QuillHtmlEditor` which makes it default to `EdgeInsets.zero`.
My assumptions is that it's due to [user agent style](https://stackoverflow.com/questions/12582624/what-is-a-user-agent-stylesheet) of the browser within the WebView, which often sets a margin of 8 px to the `body` tag but I could be wrong.

I checked this both on Android and iOS simulator and on a real iPhone (X with iOS 16.2)

**Screenshots**

<img width="253" alt="Screenshot 2023-03-09 at 14 19 32" src="https://user-images.githubusercontent.com/84321030/224037734-ae5f8bbe-e1da-4069-a197-2556323b85b0.png">

**Flutter Doctor**
```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.7.3, on macOS 13.1 22C65 darwin-arm64, locale en-DE)
[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 14.2)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2021.3)
[✓] IntelliJ IDEA Ultimate Edition (version 2022.2.3)
[✓] Connected device (5 available)
[✓] HTTP Host Availability
```

**Additional context**
If the user agent style is really responsible for this, we should have the possibility to style HTML tags by providing a style map to the quill editor widget.

**Comment by the-airbender** [comment]
Hi @gerken-tss,

Thanks for reporting the issue. 

I am able to recreate this and fixed it. The padding was not getting applied to the editor `div`, that is the reason for the extra space. 

<img src="https://user-images.githubusercontent.com/57312080/224114648-9d3fa2b9-7d65-4847-949c-3f945e4ad7d8.png" width="400" height="616">

Also,
>My assumptions is that it's due to [user agent style](https://stackoverflow.com/questions/12582624/what-is-a-user-agent->stylesheet) of the browser within the WebView, which often sets a margin of 8 px to the body tag but I could be wrong.

This is true, we also had a problem with `body` tag earlier but, it was already resolved in the older version.


This fix will be moved in the next release. I appreciate your feedback.

Thanks!

**Comment by the-airbender** [comment]
Hi @gerken-tss 

This issue is resolved, please check out the new ver [2.1.6](https://pub.dev/packages/quill_html_editor/versions/2.1.6)

Thanks :)

**Comment by the-airbender** [comment]
Thank you, will check it out.

**Comment by gerken-tss** [comment]
@the-airbender 

Should I open a separate issue for this or is this still on your agenda if it's inside a closed issue? 🙂

**Comment by the-airbender** [comment]
Hi @gerken-tss ,

I have noted it down on my local list, will fix it and deploy it in next release, planning to deploy in 2-3 days. Reopening this issue for reference. 

Thanks :)

**Comment by gerken-tss** [comment]
@the-airbender 
Excellent! Thank you for the amazingly quick response!

**Comment by the-airbender** [comment]
Hi @gerken-tss 

The hinttext padding issue is resolved and deployed it in latest version [2.1.8](https://pub.dev/packages/quill_html_editor/versions/2.1.8)

Please share your feedback.

Thanks :)
